### PR TITLE
split out deprecated structs into separate schema file

### PR DESCRIPTION
**Pull request by adeebshihadeh**

### CODE REVIEW

File: log.capnp [hunk 1]
```diff
@@ -2095,56 +2039,67 @@ struct Event {
...
+    liveUIDEPRECATED @14 :Legacy.LiveUI;
+    sensorEventDEPRECATED @4 :SensorEventData;
+    liveEventDEPRECATED @8 :List(Legacy.LiveEventData);
+
+    ethernetData @26 :List(Legacy.EthernetPacket);
+    cellInfo @28 :List(Legacy.CellInfo);
+    wifiScan @29 :List(Legacy.WifiScan);
+
+    # legacy?
+    liveMapData @62 :LiveMapData;
```
Comment (adeebshihadeh) [inline log.capnp:2085]: @HaraldSchafer these all good to deprecate?

Comment (HaraldSchafer) [reply log.capnp:2011]: yes, don't know anything about ethernetData, cellinfo and wifiScan  but others are good to go

**Comment by HaraldSchafer** [comment]
Anything blocking this merge?

**Comment by adeebshihadeh** [comment]
This isn't backwards compatible. capnproto automatically generates an id for each struct, and we have to copy that implicit id and make it explicit like we do with controlsState.

**Pull request merged**